service/src/test/java/com/ge/predix/controller/test/SubjectPrivilegeManagementControllerIT.java

```diff
@@ -124,7 +124,12 @@ public void testSubjects() throws Exception {
         // Append a list of subjects
         MockMvcContext postContext =
             TEST_UTILS.createWACWithCustomPOSTRequestBuilder(this.wac, this.testZone.getSubdomain(), SUBJECT_BASE_URL);
-        postContext.getMockMvc().perform(postContext.getBuilder().contentType(MediaType.APPLICATION_JSON)
+        postContext.getMockMvc().perform(postContext.getBuilder().contentType(MediaType.TEXT_PLAIN)
```

Comment:
Instead of changing en existing positive test into a pseudo-negative one, pull this logic out into a separate negative test that just asserts that an `HTTP 415 Unsupported Media Type` error code is returned without doing the rest of the stuff in this function.